test/libsolidity/smtCheckerTests/bmc_coverage/range_check.sol

```diff
@@ -0,0 +1,57 @@
+pragma experimental SMTChecker;
+
+contract C {
+	constructor() payable {
+		assert(tx.origin >= address(0));
```

Comment:
May have missed something. Does this PR also add constraints for `tx.origin` or is this test case for a previously added constraint that was missing a test case?

Reply:
Yes, this PR adds also type constraints for `tx.origin`which were missing before.

Reply:
okay, I looked at https://github.com/ethereum/solidity/pull/10704/files#diff-fa0f6553d9da37a75e07770a2233fabd4c32b7b91b96929b5f42c7fab0ec696cR167 and the positive diff only covered placing a constraint on `msg.value` so wasn't sure. Could you please link the lines where constraints on `tx.origin` is added? Thanks!

Reply:
You are right than in BMC only the constraint on `msg.value` has been added, but all constraints were added for CHC:
https://github.com/ethereum/solidity/pull/10704/files#diff-42bd29aaf4d31608e91e7940d97d1710c3c6bf6f860bf70f5885afe8b96f68b9